spec/index.md

```diff
@@ -1263,6 +1263,8 @@ The following sections describe various System API functions, also referred to a
     ic0.msg_cycles_accept128 : (max_amount_high : i64, max_amount_low: i64, dst : i32)
                            -> ();                                               // U Rt Ry
 
+    ic0.burn_cycles : (amount_to_burn : i64) -> (amount : i64);               // U T Rt Ry
```

Comment:
Do you also plan a 128-bit version?

Reply:
nit: We typically put the verb at the end of a system call name, so maybe this should be `ic0.cycles_burn` instead?

Reply:
I guess we *only* want the 128-bit version, no? I think we consider the 64-bit versions as deprecated.

Reply:
My reasoning was that no one will need more the max value of 64-bit cycles to burn in one go. But it is small change so I can change it to 128 if everyone agrees.

Reply:
It is changed.

Reply:
Maybe `cycles_burn128` to be consistent with the remaining function names in the 128-bit API.

Reply:
If you feel strongly about that I can change it, but I prefer it this way because we only have one `cycles_burn` API.

Reply:
I don't have a strong opinion. How about @Dfinity-Bjoern and @dsarlis ?

Reply:
I can see the arguments of both sides -- consistency with other apis but also in this case we have only the 128-bit api.

Given Björn's comment that we can consider the 64-bit versions deprecated and that one day we might only have 128-bit apis but I don't expect we'll ever go back and change their names, we might choose `cycles_burn128` for the sake of consistency.

But I'm also not feeling strongly about this, I think it's ultimately up to you guys as the spec owners.

Reply:
I would suggest consistency and use `cycles_burn128`.